### PR TITLE
Feature/#1 refactor names

### PR DESCRIPTION
**Pull request by sgirardin**
Erased all presence of beecon and also updated the package naming

### CODE REVIEW

**Review by davidantonlou — APPROVED** [review]

**Review by davidantonlou — APPROVED** [review]

File: easyContent2Metadata-platform-jar/src/log4j.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Root logger option
 log4j.rootLogger=DEBUG, Console, file
 
-logger.com.mimacom.alfresco.*=DEBUG
-log4j.com.mimacom.alfresco.*=DEBUG
+logger.com.mimacom.alfresco.*=INFO
```
Comment (wernernaehle) [inline easyContent2Metadata-platform-jar/src/log4j.properties]: we shouldn't use "mimacom", should we?

File: easyContent2Metadata-platform-jar/src/log4j.properties [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+log4j.com.mimacom.alfresco.*=INFO
```
Comment (wernernaehle) [inline easyContent2Metadata-platform-jar/src/log4j.properties]: the same about using "mimacom"

File: easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/extractor/PDFExtractor.java [hunk 3]
```diff
@@ -20,10 +19,8 @@
  */
 public class PDFExtractor implements Extractor{
 
-    private Logger logger = LoggerFactory.getLogger(this.getClass().getName());
-
     private static final String METADATA_REGION = "metadata";
-
+    private Logger logger = LoggerFactory.getLogger(this.getClass().getName());
```
Comment (wernernaehle) [inline easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/extractor/PDFExtractor.java]: Change to "private static final Logger LOGGER"

File: easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/utils/Constants.java [hunk 4]
```diff
@@ -7,16 +7,13 @@
  * Created by davidanton on 24/4/17.
  */
 public class Constants {
+    public static final String EXTRACTOR_MAPPING_TYPE_LIST = "ec2m:extractorMappingDatalist";
+    public static final QName ON_CREATE_NODE = QName.createQName(NamespaceService.ALFRESCO_URI, "onCreateNode");
+    public static final QName ON_UPDATE_CONTENT = QName.createQName(NamespaceService.ALFRESCO_URI, "onUpdateContent");
+    public static final QName ON_SET_NODE_TYPE = QName.createQName(NamespaceService.ALFRESCO_URI, "onSetNodeType");
     private static final String EXTRACTOR_MAPPING_MODEL = "http://www.mimacom.com/alfresco/beecon/1.0";
```
Comment (wernernaehle) [inline easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/utils/Constants.java]: we should change the model namespaces, but this is for other task, right?

File: easyContent2Metadata-platform-jar/src/main/resources/alfresco/module/easyContent2Metadata-platform-jar/model/easyContent2Metadata-model.xml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<model name="be:beeconModel" xmlns="http://www.alfresco.org/model/dictionary/1.0">
+<model name="ec2m:easyContent2Metadata" xmlns="http://www.alfresco.org/model/dictionary/1.0">
 
     <!-- Optional meta-data about the model -->
-    <description>Beecon Document Model</description>
+    <description>EasyContent2Metadata Document Model</description>
     <author>mimacom</author>
```
Comment (wernernaehle) [inline easyContent2Metadata-platform-jar/src/main/resources/alfresco/module/easyContent2Metadata-platform-jar/model/easyContent2Metadata-model.xml:6]: we could use "sidawe" as author :-) or something different

**Review by wernernaehle — COMMENTED** [review]
Needs work

File: easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/extractor/PDFExtractor.java (hunk 3 above)
Comment (sgirardin) [reply easyContent2Metadata-platform-jar/src/main/java/org/alfresco/extension/ec2m/extractor/PDFExtractor.java]: good point. We will need to to look with David for his part. But I think on the long term I haev found a more interesting solution: http://www.yegor256.com/2014/05/23/avoid-java-static-logger.html

File: easyContent2Metadata-platform-jar/src/main/resources/alfresco/module/easyContent2Metadata-platform-jar/model/easyContent2Metadata-model.xml (hunk 5 above)
Comment (sgirardin) [reply easyContent2Metadata-platform-jar/src/main/resources/alfresco/module/easyContent2Metadata-platform-jar/model/easyContent2Metadata-model.xml:6]: good point we can discuss about it for a futur commit

File: easyContent2Metadata-platform-jar/src/log4j.properties (hunk 1 above)
Comment (sgirardin) [reply easyContent2Metadata-platform-jar/src/log4j.properties]: Good catch

File: easyContent2Metadata-platform-jar/src/log4j.properties (hunk 2 above)
Comment (sgirardin) [reply easyContent2Metadata-platform-jar/src/log4j.properties]: Good catch

**Pull request merged**